doc/nrf/drivers/wifi/nrf700x/nrf700x.rst

```diff
@@ -1,36 +1,36 @@
 .. _nrf700x_wifi:
 
-nRF700X Wi-Fi driver
-####################
+nRF70 Series Wi-Fi driver
+#########################
 
 .. contents::
    :local:
    :depth: 2
 
-This driver implements the Wi-Fi® protocol for the nRF700X FullMAC series of devices.
+This driver implements the Wi-Fi® protocol for the nRF70 series FullMAC series of devices.
```

Comment:
```suggestion
This driver implements the Wi-Fi® protocol for the nRF70 Series FullMAC series of devices.
```
Just title and first instance needs "Series" capitalized. The rest are fine as is.